slack-archive/src/main/java/juja/microservices/slack/archive/service/ArchiveService.java

```diff
@@ -1,12 +1,15 @@
 package juja.microservices.slack.archive.service;
 
-import juja.microservices.slack.archive.model.ChannelDTO;
-import juja.microservices.slack.archive.model.Message;
-import juja.microservices.slack.archive.model.MessagesRequest;
+import juja.microservices.slack.archive.model.dto.ChannelDTO;
+import juja.microservices.slack.archive.model.dto.MessagesRequest;
+import juja.microservices.slack.archive.model.entity.Message;
 
 import java.util.List;
 
 public interface ArchiveService {
```

Comment:
Maybe is better naming ChannelService and extract some messages logic to another service

Reply:
Yes, it is a good idea and in this case I will do separate repositories for channel and messages

Reply:
done!